PLUGINS.md

```diff
@@ -219,8 +219,8 @@ title will be the value that you returned from `plugin_start3`. Use widgets
 from EDMC's myNotebook.py for the correct look-and-feel. You can be notified
 when the settings dialog is closed so you can save your settings.
 
-You can use `set()`, `get()` and `getint()` from EDMC's `config.config` object
-to retrieve your plugin's settings in a platform-independent way.
+You can use `set()` and `get_$type()` (where type is one of: `int`, `bool`, `str`, `list`) from EDMC's 
```

Comment:
Add something to be explicit about what the old methods were and that they're deprecated.  People won't necessarily notice the code logged deprecation warnings.